src/etc/inc/certs.inc

```diff
@@ -281,6 +281,42 @@ function ca_inter_create(& $ca, $keylen, $lifetime, $dn, $caref, $digest_alg = "
 	return true;
 }
 
+function ca_trustedcerts_generate() {
+	global $config, $g;
+	if (isset($config['system']['developerspew'])) {
+		$mt = microtime();
+		echo "ca_trustedcerts_generate() being called $mt\n";
+	}
+
+	mwexec("/bin/mkdir -p {$g['etc_path']}/ssl/certs");
+	/* Remove all certs in the directory (but leave symlinks in place to allow power users to add trusted certs outside the Certificate Manager) */
+	mwexec("/usr/bin/find {$g['etc_path']}/ssl/certs -type f -exec /bin/rm '{}' ';'");
+
+	if (is_array($config['ca'])) {
+		foreach ($config['ca'] as & $ca) {
+			if ($ca['trusted']) {
```

Comment:
And along with the other issets this would be an isset test also. `if (isset($ca['trusted']))`